form.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+    function func(String $text, int $size, &$valid, &$errors){
+        $tmp = $_POST["$text"];
+        if(!$tmp){
+            $errors["$text"] = "Полето е задължително."; 
+        } elseif(strlen($tmp) > $size ){
+            $errors["$text"] = "Полето има максимална дължина $size символа.";
+        } else {
+            $valid[$text] = $tmp;
+        }        
+    }
+
+    $valid = array();
+    $errors = array();
+    
+    if( $_POST ){
+        func("title", 150, $valid, $errors);
+        func("teacher",200, $valid, $errors);
```

Comment:
I feel here we could apply autoformatting with visual code: `Shift+Alt+F` 
Source: https://dzone.com/articles/setting-custom-shortcuts-in-visual-studio-code